content/recipes/introduction/priorization.md

```diff
@@ -0,0 +1,207 @@
+
+**An *in-house* prioritisation scheme can be established by FAIR experts for selection of relevant projects for the FAIRification process.** This involves a two-step approach:
+<ol>
+	<li>Identification of certain priority disease areas, such as COVID-19, neurodegenerative diseases, cardiovascular diseases, etc.</li>
+	<li>Prioritising projects falling into these priority areas compared to others.</li>
+</ol>
+
+#### 2. Prioritisation based on timeline
+
+The process for FAIRification could be done either to the data present in a project or to a processing pipeline that is part of the project. Due to the interdependence of the data or process with the project, the time range at which a project runs is an important criteria to understand the availability of the data or pipeline. Taking a deeper look into the data dependency for the FAIRification process, a project can be divided into three stages of development: early stage, middle stage, and end stage.
+- A project is in early stages of development when the data requirements for the given project are being listed and simultaneously being collected. At this stage, the data availability is the lowest, and it is easier to design a data model, choose ontologies etc. that ensures creation of FAIR data by design'.
+- In the middle stage, all the data relevant for the given project has been collected and now needs to be standardised for downstream tasks such as for predictions using machine learning approaches. This is a stage where there is maximum availability of data. 
+- Lastly, in the end stage, data has either been deposited or handed over to the respective heads and the project is near its termination. Furthermore, it is at this stage that the risk  of organisation restructuring is highest, with staff reassignment or departure. Hence, starting a FAIRification process at a project  end stage could be the least favourable as this could place large demands in time and resources on key personnel, such as for corresponding data owners and data generators, at a time of effort wind down.
+
+**Hence, the best time of engaging a project into the FAIR pipeline would be dependent on the retrospective and prospective aspect of the project.** The best time to FAIRify prospective projects is the early stage. This stage is the best time to define and lay out metadata standards that need to be followed during the project trajectory. As a result, it eliminates many of the downstream logistical and financial problems. On the other hand, the FAIRification of retrospective projects is in the middle stage. This is because it is a data rich stage and contact with both data owners and generators could be established for better understanding of the data.
```

Comment:
```suggestion
**Hence, the best time of engaging a project into the FAIR pipeline would be dependent on the retrospective and prospective aspects of the project.** `The best time to FAIRify prospective projects is the early stage`. This stage is the best time to define and layout metadata standards that need to be followed during the project's trajectory. As a result, it eliminates many of the downstream logistical and financial problems. On the other hand, the FAIRification of retrospective projects is in the middle stage. This is because it is a data-rich stage and contact with both data owners and generators could be established for a better understanding of the data.
```